app/Http/Controllers/Admin/ProductsController.php

```diff
@@ -34,7 +37,32 @@ public function index(Request $request)
      */
     public function create()
     {
-        return view('admin.product.create');
+        $category = Category::all();
+        return view('admin.product.create', compact('category'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CreateProductRequest  $request)
+    {
+        $product = Product::create($request->all());
```

Comment:
ở đây nên thêm try catch và db transaction vào em, để đảm bảo toàn vẹn dữ liệu 
https://laravel.com/docs/5.6/database#database-transactions